src/playasophy/wonderdome/mode/strip_eq.clj

```diff
@@ -0,0 +1,75 @@
+(ns playasophy.wonderdome.mode.strip_eq
+  (:require
+    [playasophy.wonderdome.geometry.sphere :as sphere]
+    [playasophy.wonderdome.mode.core :as mode]
+    [playasophy.wonderdome.util.color :as color]
+  )
+)
```

Comment:
Typically in lisps you close the s-expression with parenthesis on the last line of content, so in this case line 5 would end in `]))`.